sklearn/calibration.py

```diff
@@ -25,6 +27,337 @@
 from .svm import LinearSVC
 from .model_selection import check_cv
 from .metrics.classification import _check_binary_probabilistic_predictions
+from .metrics.ranking import precision_recall_curve, roc_curve
+from .utils.multiclass import type_of_target
+
+
+class CutoffClassifier(BaseEstimator, ClassifierMixin, MetaEstimatorMixin):
+    """Decision threshold calibration for binary classification
+
+    Meta estimator that calibrates the decision threshold (cutoff point)
+    that is used for prediction. The methods for picking cutoff points make use
+    of traditional binary classification evaluation statistics such as the
+    true positive and true negative rates and F-scores.
+
+    If cv="prefit" the base estimator is assumed to be fitted and all data will
+    be used for the selection of the cutoff point. Otherwise the decision
+    threshold is calculated as the average of the thresholds resulting from the
+    cross-validation loop.
+
+    Parameters
+    ----------
+    base_estimator : obj
+        The classifier whose decision threshold will be adapted according to
+        the acquired cutoff point. The estimator must have a decision_function
+        or a predict_proba
+
+    method : str
+        The method to use for choosing the cutoff point
+
+        - 'roc', selects the point on the roc curve that is closest to the
+        ideal corner (0, 1)
```

Comment:
please use a hanging indent to format the list.

An alternative syntax that might be helpful is the ReST definition list:

```
`roc`
    selects the point on the ...
    ideal corner ...
`f_beta`
    selects a ...
```